net/nfs.c

```diff
@@ -22,13 +22,20 @@
  * possible, maximum 16 steps). There is no clearing of ".."'s inside the
  * path, so please DON'T DO THAT. thx. */
 
+/* NOTE 4: NFSv3 support added by Guillaume GARDET, 2016-June-20.
+ * NFSv2 is still used by default. But if server does not support NFSv2, then
+ * NFSv3 is used, if available on NFS server. */
+
```

Comment:
Please keep backported upstream commits separate with reference to upstream commit id. You can use `git cherry-pick -x` for that.